src/track/text-style.js

```diff
@@ -168,11 +166,9 @@ class TextStyle {
   }
 
   /**
-   * Font size, such as 1, 2, 3...
+   * Font scale, such as 1, 2, 3...
    * @type {number}
    */
-  fontSize: string = '100%';
-
   fontScale: number = 1;
```

Comment:
since the jsdoc has been modified, please rebuild api.md by running "yarn run docs:generate"